report/renderer/performance-category-renderer.js

```diff
@@ -312,18 +312,18 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       ...filterableMetrics,
     ]);
     for (const metric of filterChoices) {
-      const elemId = `metric-${metric.acronym}`;
-      const radioEl = this.dom.createChildOf(metricFilterEl, 'input', 'lh-metricfilter__radio');
-      radioEl.type = 'radio';
-      radioEl.name = 'metricsfilter';
-      radioEl.id = elemId;
+      const {acronym} = metric;
 
       const labelEl = this.dom.createChildOf(metricFilterEl, 'label', 'lh-metricfilter__label');
-      labelEl.htmlFor = elemId;
       labelEl.title = metric.result && metric.result.title;
-      labelEl.textContent = metric.acronym || metric.id;
+      labelEl.textContent = acronym || metric.id;
+      labelEl.dataset.acronym = acronym;
+
+      const radioEl = this.dom.createChildOf(labelEl, 'input', 'lh-metricfilter__radio');
```

Comment:
so this is reversing what was done here for tab a11y: https://github.com/GoogleChrome/lighthouse/pull/12552

also I noticed that `radioEl.hidden = 'true'` was a thing before my change and is missing here. is it not needed again?

Reply:
> also I noticed that radioEl.hidden = 'true' was a thing before my change and is missing here.

i didnt add these styles just now but... maybe it was double hidden?

![image](https://user-images.githubusercontent.com/39191/126727564-58819605-bffa-488e-92d9-f2b0abcbb6ea.png)


i'm nearly certain the hidden affected the tababaility of this, but after trying a few things, i'm mostly thinking going back to id/for is most straightforward